### PR TITLE
Initialize components

### PR DESCRIPTION
**Pull request by why-a-en**
# Branch changes

- Setup a new React application using Create React App
- Created a directory for reusable components: /components.
- Installed React Router V6
- Setup Router inside of App.js
- Added 2 routes in the application: Books and Category
- Created components inside of the /components directory:
- book.js contains individual book states, as well as a button to delete the book
- bookList.js contains a list to render individual books
- addBook.js contains a form to create new books
- navigation.js contains navigation to link to the 2 routes created earlier

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,20 @@
-import logo from './logo.svg';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Books from './components/books';
+import Category from './components/categories';
+import Navigation from './components/navigation';
 import './App.css';
```
Comment (ABDULALI3468) [inline src/App.js]: Please consider removing this line as there is no `App.css` and is causing the Error and not allowing the app to run 💯 

image

File: src/components/booksList.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+const BooksList = () => (
+  <div className="booksList">
+    <Book />
+    <Book />
+    <Book />
+  </div>
+);
+
+export default BooksList;
```
Comment (ABDULALI3468) [inline src/components/booksList.js]: Please consider creating a state object in your bookList component and map over the state array and pass each object to the `book component` as props and the book component render these values which you passed as props to meet the project requirements!

### CODE DEMONSTRATION:-


```
import Book from './book';

const BooksList = () => {
  const list = [
    {
      category: 'Science Fiction',
      title: 'Dune',
      author: 'Frank Herbert',
      progress: '65%',
      chapter: 'Chapter 9',
    },
  ];

  const {
    category, title, author, progress, chapter,
  } = list[0];

  return (
    <div>
      <Book
        category={category}
        title={title}
        author={author}
        progress={progress}
        chapter={chapter}
      />
    </div>
  );
};

export default BooksList ;

```


### NOTE:-

It is a general theme you can have more than one object in the array or something else but need to pass it in props to child components 💯

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @anbehindY , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights
☑️Created basic project setup! 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

In your README file there is no Acknowledgments section please consider adding this section to make your project look more professional.

image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @anbehindY , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good use of React components :white_check_mark: 
- [x] Good file and source code organization :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark: 
- [x] Implemented all the changes from the previous reviewer :white_check_mark:    

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**